packages/cozy-clisk/src/bridge/bridgeInterfaces.ts

```diff
@@ -0,0 +1,51 @@
+import { MethodsType } from "post-me";
+
+/**
+ * All bridges are supposed to implement this interface
+ */
+
+export interface Bridge {
+  remoteHandle: Function;
+  localHandle: Function;
+  init(initObject: InitObject ): Promise<void>;
+  call(method: string, ...args: Array<any>): Promise<any>;
+  emit(eventName: string, ...args: Array<any>);
+  addEventListener(remoteEventName: string, listener: Function);
+  removeEventListener(remoteEventName: string, listener: Function);
+
+}
+
+export interface InitObject { 
+  //The object which will contain the exposed method names
+  root?: Object
+  // The list of method names of the root object, which will be exposed via the post-me interface to the content script
+  exposedMethods?: MethodsType
+  // The list of method names of the root object, which will be call on given event name via the post-me interface to the content script
+  listenedEventsNames?: Array<string>
+  // Reference to the webview obect containing the content script
+  webViewRef?: Object 
+}
+
+
+
+/**
+ * All messengers are supposed to implement this interface
+ *
+ * @interface
+ */
+export interface MessengerInterface {
```

Comment:
Now that you converted this MessengerInterface to a real TS interface it is not possible to import it in js code I think. When you look at the /dist/bridge/bridgeInterfaces.js it builds as an empty file.

Should we migrate flagship app files using theses interfaces to TS ?

Reply:
I think we should because we need some changes present here for the launcher to work.
Ex : changes in addData 
```
- import CozyClient from 'cozy-client/types/CozyClient'
...
- * @param {CozyClient} options.client : CozyClient instance
+ * @param {import('cozy-client/types/CozyClient').default} options.client CozyClient instance
```

Reply:
I'll split the PR in order to quickly release a fix.

Reply:
> Should we migrate flagship app files using theses interfaces to TS ?

@doubleface To me, this is not needed. We can do it, but we can also do differently: 
- You can remove the `extends` keyword for the class' on the flagship side
- Return the correct type (jsdoc) for these classes 

In all cases, we need to rewrite the classes, since `clisk` should not publish any code related to the interface.